AppCenter/AppCenter/Internals/Ingestion/MSHttpIngestion.m

```diff
@@ -96,13 +96,13 @@ - (id)initWithBaseUrl:(NSString *)baseUrl
 
 #pragma mark - MSIngestion
 
+- (BOOL)isReadyToSend {
```

Comment:
You don't need to override this in `MSOneCollectorIngestion`?

Reply:
No, because target tokens are stored in the logs itself and we don't need to make additional actions to prepare ingestion for sending.